locations/spiders/ihg_hotels.py

```diff
@@ -1,15 +1,14 @@
-import scrapy
+from scrapy.spiders import SitemapSpider
 
-from locations.linked_data_parser import LinkedDataParser
-from locations.open_graph_parser import OpenGraphParser
+from locations.structured_data_spider import StructuredDataSpider
 
 
-class IHGHotelsSpider(scrapy.spiders.SitemapSpider):
+class IHGHotelsSpider(SitemapSpider, StructuredDataSpider):
     name = "ihg_hotels"
     allowed_domains = ["ihg.com"]
-    sitemap_urls = ["https://www.ihg.com/bin/sitemapindex.xml"]
-    sitemap_rules = [(r".*/us/en/.*/hoteldetail$", "parse")]
-    download_delay = 0.5
+    sitemap_urls = ["https://www.ihg.com/bin/sitemap.holidayinnresorts.en.hoteldetail.xml"]
```

Comment:
We want to keep this at `https://www.ihg.com/bin/sitemapindex.xml` so that it fetches all the other brands, too.